packages/frontend/components/hooks/useEthPrice.ts

```diff
@@ -0,0 +1,17 @@
+import { Fetcher, Pair, Route, Token, WETH } from "@uniswap/sdk";
+
+export const useEthPrice = async (provider: any): Promise<number> => {
+  // Get the price of ETH based on ETH/DAI trading pair from Uniswap SDK
```

Comment:
You can move this comment above the function declation.

Check others hooks/component to copy the "comment style"